lib/vrl/core/Cargo.toml

```diff
@@ -1,17 +1,22 @@
 [package]
-name = "vrl"
+name = "vrl-core"
 version = "0.1.0"
 authors = ["Vector Contributors <vector@timber.io>"]
 edition = "2021"
 publish = false
 
 [dependencies]
-compiler = { package = "vrl-compiler", path = "../compiler" }
-diagnostic = { package = "vrl-diagnostic", path = "../diagnostic" }
-parser = { package = "vrl-parser", path = "../parser" }
```

Comment:
Tl;dr: All the other changes are a consequence of this 😄

Reply:
_that's a lotta changes_